lib/pubsuffix-psl.ts

```diff
@@ -58,11 +58,12 @@ export function getPublicSuffix(
 
   if (
     allowSpecialUseDomain &&
-    typeof topLevelDomain === 'string' &&
+    topLevelDomain !== undefined &&
```

Comment:
Why this switch?

Reply:
it was either a linter or IDE suggestion